### DISCUSSION TITLE
My GitHub contributions are not showing on my contribution graph.

### DISCUSSION THREAD

**Discussion by AlexETay7** [community/community#126650] [Other Feature Feedback, Questions, & Ideas]
Bug

### Body

I have tried to commit from multiple different repos of mine and am unable to get the contribution graph to change. I am committing from the main branch and am not doing anything different than I have in the past. The only repo that shows up on the graph when I change it is my README repo... please help.

**Comment by noradodev** [comment]
Hey there!  It seems like your GitHub contributions aren't showing up on your graph, even though you've been putting in the effort. No worries at all, these things can be tricky sometimes!

There could be a few reasons why this might be happening:

1.     Email Mix-up: Maybe the email you're using for committing is different from the one linked to your GitHub account. You can check this on your computer with a special code (git config user.email). If they're not the same, you can just update your computer's code or add the commit email to your GitHub settings.
2.     Privacy Matters: Only contributions you make to public repositories show up on your public contribution graph. So, if you've been working on something top-secret (like a surprise for someone special!), those commits won't be there.
3.     Helping Hands, Not Forks: Contributing to a copy of someone else's project (called a fork) won't show up on your main graph. To get credit, your changes need to be merged into the original project.
4.     Patience is Key: Sometimes, GitHub just needs a little time to catch its breath and update your graph. Give it a while after you commit and see if it shows up!
5.     Calendar to the Rescue: Even if the graph isn't showing everything right now, your contributions might still be recorded! Check the contribution calendar on your profile for a more detailed view.

If you've tried all this and your graph is still being a bit stubborn, don't hesitate to reach out to GitHub support. They can sometimes give it a little nudge to refresh and show off all your hard work.

**Reply by AlexETay7** [reply]
Hello, thanks for the help. Unfortunately I have done all these things, and I have waited 24 hours to see if the contributions will show up on my calendar graph and nothing is showing up. How can I reach out to GitHub support? I do not pay for GitHub so I don't know how to directly contact them.

**Reply by ebndev** [reply]
Hi @AlexETay7, thanks for being a part of the GitHub Community!

I've moved your question to our Profile category, where you can see many discussions in the past that have had the same issue. Please see if any of the answered discussions [here](https://github.com/orgs/community/discussions/categories/profile?discussions_q=category%3AProfile+contributions+not+showing) will help resolve your issue 😄 

P.S GitHub Support does not typically provide support for contributions. Thanks for understanding!

**Comment by davevad93** [comment]
Hi @AlexETay7 ,

I've taken a look at your last contributions in the [Weather.](https://github.com/AlexETay7/Weather.) and [Differential-Leveling-for-Civil-Engineers](https://github.com/AlexETay7/Differential-Leveling-for-Civil-Engineers) repos, the thing is that those commits were made by a different account, as you can see in the screenshot I'm going to show you below:

![1111](https://github.com/community/community/assets/94308234/9315d439-c4e0-4073-856c-0b1bab77d0db)

![2222](https://github.com/community/community/assets/94308234/43ddd115-8c00-411f-9a2a-e707cf45e224)
 
That's the reason why your latest contributions aren't showing up in your profile which is probably due to some configuration issue, so take a look at your git configuration files and make sure the email is the same as the one associated with your account.
